components/Alert/alert.css

```diff
@@ -0,0 +1,71 @@
+
+@import "/reset.css";
+.alert-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```

Comment:
if these code repeat you can put it in the utility class

Reply:
yes done